packages/liveblocks-client/src/AuthToken.ts

```diff
@@ -135,10 +136,13 @@ export function parseRoomAuthToken(
     } = data;
     return token;
   } else {
-    throw new Error("Authentication error: invalid room auth token");
+    throw new Error(
+      "Authentication error: we expected a room token. Hint: if you are using a callback, ensure the room is passed when creating the token. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClient"
+    );
   }
 }
 
+// TODO: unused in all packages & not accessible from @liveblocks/client users, remove?
```

Comment:
Ha! Great catch, I think you're right. It's good to also always check usage of these APIs in https://github.com/liveblocks/liveblocks-cloudflare and https://github.com/liveblocks/liveblocks.io, because those might also import these APIs using `import { parseAuthToken } from "@liveblocks/client/internal"`, but I just checked them and they're also not using this anymore either!

I think what happened is that at the end of some refactorings, we ended up with only usages of `parseRoomAuthToken`, and never this one anymore, and I forgot to clean this one up. I think it would indeed be safe to remove this — nice! ✨🧽✨

Reply:
Oh you're right, I'll do a sanity check in those codebases as well from now on. Will clean it up then :)